css/properties/content-visibility.json

```
@@ -0,0 +1,54 @@
+{
+  "css": {
+    "properties": {
+      "content-visibility": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/content-visibility",
+          "support": {
+            "chrome": {
+              "version_added": "85"
+            },
+            "chrome_android": {
+              "version_added": "85"
+            },
+            "edge": {
+              "version_added": "85"
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "71"
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
```

There's only one implementation for this so far, so this needs to be `true`.

```suggestion
            "experimental": true,
```

---

**Opera 71** (released on Sep 15, 2020) and **Chrome 85** (released on Aug 25, 2020).

---

Opera and Chrome are both Chromium/Blink, so that's only one engine implementing the feature. I stand by my original suggestion.